### PR TITLE
SIG Big Data: improved and updated overview of resources regarding SIG activities.

### PR DESCRIPTION
**Pull request by berndfo**
For SIG Big Data, updating the document outlining current SIG activities. Information is separated into status and activities for each project.

### CODE REVIEW

File: sig-big-data/resources.md [hunk 1]
```diff
@@ -1,15 +1,61 @@
 # Resources
 
+## Kubernetes Integration status by Big Data product
```
Comment (liyinan926) [inline sig-big-data/resources.md]: Can we use the capital case for the first letter of each word consistently?

File: sig-big-data/resources.md [hunk 2]
```diff
@@ -1,15 +1,61 @@
...
+That work was done after the [Spark on Kubernetes original Design Proposal](https://docs.google.com/document/d/1_bBzOZ8rKiOSjQg78DXOA3ZBIo_KkDJjqxVuq0yXdew/edit#)
+in the [apache-spark-on-k8s git repo](https://github.com/apache-spark-on-k8s/spark).
+
+##### Activities 
+
+Enhancements are underway, with a good overview given [in this blog post](https://databricks.com/blog/2018/09/26/whats-new-for-apache-spark-on-kubernetes-in-the-upcoming-apache-spark-2-4-release.html).
+
+* Work is underway for Spark 2.4 to improve support and integration with HDFS.
+  * Design Document: [How Spark on Kubernetes will access Secure HDFS](https://docs.google.com/document/d/1RBnXD9jMDjGonOdKJ2bA1lN4AAV_1RwpU_ewFuCNWKg/edit#heading=h.verdza2f4fyd)
+* shuffle service design
```
Comment (liyinan926) [inline sig-big-data/resources.md]: `shuffle` -> `Shuffle`.

File: sig-big-data/resources.md [hunk 3]
```diff
@@ -1,15 +1,61 @@
...
+[Apache Hadoop HDFS](https://hadoop.apache.org/hdfs) is a distributed file system, the persistence layer for Hadoop.
+
+##### Status
+
+TODO, e.g. "No release yet."
+
+##### Activities
+
 * [Data Locality Doc](https://docs.google.com/document/d/1TAC6UQDS3M2sin2msFcZ9UBBQFyyz4jFKWw5BM54cQo/edit)
 * [External Repository](https://github.com/apache-spark-on-k8s/kubernetes-HDFS)
```
Comment (liyinan926) [inline sig-big-data/resources.md]: `External Repository` -> `Helm Charts`.

File: sig-big-data/resources.md (hunk 1 above)
Comment (berndfo) [reply sig-big-data/resources.md]: @liyinan926 pushed change in latest commit. looks consistent now?

File: sig-big-data/resources.md (hunk 2 above)
Comment (berndfo) [reply sig-big-data/resources.md]: done.

File: sig-big-data/resources.md (hunk 3 above)
Comment (berndfo) [reply sig-big-data/resources.md]: clarified the link description, included Helm chart nature of the repo.

**Pull request merged**